### PR TITLE
Tell about the release cycles.

### PR DESCRIPTION
**Pull request by omoerbeek**
### Short description

If this table format is OK, I'll add the auth one.

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [X] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/recursordist/docs/appendices/EOL.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
 End of life statements
-======================
+=====================
```
Comment (zeha) [inline pdns/recursordist/docs/appendices/EOL.rst:2]: did this remove one actually required =?

Comment (omoerbeek) [reply pdns/recursordist/docs/appendices/EOL.rst]: yes, tx

File: docs/appendices/EOL.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 End of life statements
 ======================
 
+We aim to have a release every six months.
+The latest and previous release receive correctness, stability and security updates.
+The release before that gets critical security updates only.
```
Comment (zeha) [inline docs/appendices/EOL.rst:6]:
```suggestion
The second-to-last release before that gets critical security updates only.
```

File: docs/appendices/EOL.rst [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+Older releases are marked end of life.
```
Comment (zeha) [inline docs/appendices/EOL.rst:7]:
```suggestion
Older releases are marked end of life and receive no updates at all.
```

File: docs/appendices/EOL.rst [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
+
```
Comment (zeha) [inline docs/appendices/EOL.rst:8]:
```suggestion
Pre-releases do not receive immediate security updates.

```

if this is true?

File: docs/appendices/EOL.rst (hunk 2 above)
Comment (omoerbeek) [reply docs/appendices/EOL.rst:6]: I find second-to-last always confusing. To me it means the one before last here.

**Pull request merged**